content/en/observability_pipelines/production_deployment_overview/_index.md

```diff
@@ -1,42 +1,188 @@
 ---
-title: Production Deployment Overview
+title: Deployment Design and Principles
 kind: Documentation
 ---
 
 ## Overview
 
-This article walks you through architectural best practices for deploying Observability Pipelines Worker (OPW) into production environments. It goes through [guidelines](#guidelines) for common deployment mistakes, the different Observability Pipelines Worker [deployment roles](#deployment-roles), and an overview of the three [reference architectures](#reference-architectures). 
+When you start deploying Observability Pipelines Worker into your infrastructure, you may run into questions such as: 
 
-## Guidelines
+- Where should the Observability Pipelines Worker be deployed within the network?
+- How should the data be collected?
+- Where should the data be processed?
 
-### Start with one aggregator
+This guide walks you through what to consider when designing your Observability Pipelines Worker architecture, specifically these topics:
 
-Most users of Observability Pipelines Worker have complex production environments with many clusters, services, and observability tools. A common mistake is to plan multiple Observability Pipelines Worker deployments instead of starting with one. Datadog recommends to start by deploying a single Observability Pipelines Worker aggregator and onboarding one service at a time. This allows you to get set up quickly and gain experience operating Observability Pipelines Worker before adopting more complex deployment strategies. 
+- [Networking](#networking)
+- [Collecting data](#collecting-data)
+- [Processing data](#processing-data)
+- [Buffering data](#buffering-data)
+- [Routing data](#routing-data)
 
- ## Deployment roles
+## Networking
 
- The Observability Pipelines Worker can be deployed anywhere in your infrastructure as an agent or an aggregator.
+The first step to architecting your Observability Pipelines Worker deployment is understanding where Observability Pipelines Worker fits within your network and where to deploy it.
 
- ### Agent role
+### Working with network boundaries
 
- As an agent, the Observability Pipelines Worker (OPW) is deployed on each individual node, shifting data collection and processing to the edge:
+When deploying the Observability Pipelines Worker as an aggregator, it should be deployed within your network boundaries to minimize egress costs. Ingress into the Observability Pipelines Worker should never travel over the public internet. Therefore, Datadog recommends starting with one aggregator per region to keep things simple.
```

Comment:
Let's change the beginning to: "Because Observability Pipelines Worker is deployed as an aggregrator..."